package.json

```diff
@@ -93,16 +93,16 @@
     "karma-sourcemap-loader": "^0.3.7",
     "karma-webpack": "^2.0.3",
     "phantomjs-prebuilt": "^2.1.14",
-    "primeng": "^4.3.0",
-    "rxjs": "^5.2.0",
+    "primeng": "^5.2.7",
```

Comment:
When looking at their README I saw that they upgraded to Angular 6, but once I upgraded I discovered that it's not quite in the latest release. I can revert this if you'd like :)

Reply:
I see that at this moment, a new version is available : `6.0.1`, can you upgrade to this version and test if all works?